tests/modules/nf-core/mmseqs/easysearch/nextflow.config

```
@@ -0,0 +1,8 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
```

A small suggestion to clean up the output test.yml 
```suggestion
    withName: MMSEQS_CREATEDB_TARGET {
        publishDir = [ enabled: false ]
    }

```